prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/ConstantResolverTests.kt

```diff
@@ -113,6 +115,29 @@ class ConstantResolverTests {
         assertThat(result).isNull()
     }
 
+    @Test
+    fun `test fhir path resolver multiple values`() {
+        val integerValue = 99
+        val stringValue = "Ninety-Nine"
+
+        mockkObject(FhirPathUtils)
+        every { FhirPathUtils.evaluate(any(), any(), any(), any()) } returns
+            listOf<Base>(StringType(stringValue), StringType(integerValue.toString()))
+
+        val constants = sortedMapOf("const1" to "'value1'") // this does not matter but context wants something
```

Comment:
You could just say CustomContext()

Reply:
I can't because I need it to get through the first part of the `resolveConstant` function, the part that calculates "constantValue". If I don't set anything, then the code we are trying to test does not run. This tells me that this function does two separate things and should be broken up to be more cohesive, but I didn't find it egregious enough to warrant a refactor/rewrite. The other tests cover the other cases and the other parts of the function. 

The code coverage report is good, tests all the lines of the class we are testing.